docs/getting-started/use-cases/computervision/annotation/cvat/cvat_quick_guide.md

```diff
@@ -139,4 +143,4 @@ Onepanel’s CVAT supports a feature to pre-annotate images for common objects.
 
   ![Inference Output](/img/cvat_inference_output.png)
 
-If you need help or have questions regarding CVAT, feel free to reach out on our [Slack channel](https://join.slack.com/t/onepanel-ce/shared_invite/zt-eyjnwec0-nLaHhjif9Y~gA05KuX6AUg).
\ No newline at end of file
+If you need help or have questions regarding CVAT, feel free to reach out on our [Slack channel](https://join.slack.com/t/onepanel-ce/shared_invite/zt-eyjnwec0-nLaHhjif9Y~gA05KuX6AUg).
```

Comment:
This should be linking to GitHub discussions and not Slack.

If you need help or have questions regarding CVAT, feel free to reach out in [GitHub discussions](https://github.com/onepanelio/core/discussions).